runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -1175,8 +1181,8 @@ public CorfuRuntime setHoleFillingDisabled(boolean disable) {
      */
     @Deprecated
     public CorfuRuntime setNumCacheEntries(long numCacheEntries) {
```

Comment:
Is it possible to change name of set method as well? setMaxCacheEntries?

Reply:
This is deprecated, removed it.